### PR TITLE
Support tumbleweed

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

**Comment by boblmartens** [comment]
Thank you!

File: spec/unit/os_inspector_spec.rb [hunk 1]
```diff
@@ -142,6 +142,21 @@
...
+    it "is able to recognize the rolling openSUSE release Tumbleweed" do
+      FakeFS::FileSystem.clone("spec/data/os/openSUSETumbleweed/etc/os-release",
+        "/etc/os-release")
+
+      expect(inspector).to receive(:get_arch).and_return("x86_64")
+
+      inspector.inspect(filter)
+
+      expect(description.os.name).to eq "openSUSE Tumbleweed"
+      expect(description.os.version).to include(" (Tumbleweed)")
```
Comment (aduffeck) [inline spec/unit/os_inspector_spec.rb]: Why don't we match the complete version string instead?

File: man/machinery-build.1.md [hunk 2]
```diff
@@ -49,20 +49,6 @@ built image. The list of filters is shown at the beginning of the build.
   * All repositories in the system description must be accessible from the
     build machine on which `machinery build` is called.
 
-### BUILD SUPPORT MATRIX
```
Comment (aduffeck) [inline man/machinery-build.1.md]: Can't we keep this section? It is helpful to the users imo...

File: plugins/inspect/os_inspector.rb [hunk 3]
```diff
@@ -92,6 +92,11 @@ def get_os_from_os_release
     end
     if result["pretty_name"]
       result["pretty_name"] = strip_arch_from_name(result["pretty_name"])
+
+      # remove always changing Tumbleweed version from pretty name
+      if result["pretty_name"].include?("Tumbleweed")
```
Comment (aduffeck) [inline plugins/inspect/os_inspector.rb]: I'd prefer to have make the ckeck a bit more specific, e.g. 

```
if result["pretty_name"] =~ /^openSUSE.*Tumbleweed$/
```

or something like that.

**Pull request closed without merging**